.gitignore

```diff
@@ -46,4 +46,6 @@ dist/
 
 ui/npm-debug.log
 
-Gemfile.lock
\ No newline at end of file
+Gemfile.lock
+
+**/.idea
```

Comment:
@leecalcote is it necessary to add IDE specific files in .gitignore?

Reply:
Great question. If we don't, it's inevitable that someone's .vscode or .idea IDE / project settings make their way into a PR or two. If there are standard conventions that we consider those contributing to the project would benefit from as template settings for their IDE, we may want to provide those - - doing so in the CONTRIBUTING.md or similar location, but not as a versioned file (as part of the repos' code, because personalized changes that individuals make would then be lost).

Reply:
I agree with @leecalcote . It was kinda inconvenient that I had to remove the .idea folder anytime I pushed some changes. Would be nice to have it in the .gitignore; I'm a fan of intelliJ based editors like Goland!

Reply:
LGTM